app/views/generic_files/_subject_modal.html.erb

```diff
@@ -0,0 +1,41 @@
+<div class="modal-div">
+<!-- Modal -->
```

Comment:
I noticed that both the erb files are missing from this commit.. adding those now